examples/src/utils.cpp

```diff
@@ -34,6 +34,18 @@ static SCREEN* s_Screen = nullptr;
 #	include <iostream>
 #endif // USE_CURSES
 
+int _getch()
```

Comment:
This method is technically not equivalent to getch, as it alters the return value.
Either rename this method to something more semantically accurate, or just use a #define over getch/wgetch.

Reply:
@christophe-calmejane can you provide name you will like? don't have time to guess in dozens of commits

Reply:
I can't seem to find the comment I posted earlier with the name of the preprocessor macro, but just wrapping the getch call with a preprocessor macro is enough, this is just a sample file, no need to waste too much time on it. The name of the macro was simply **GETCH** IIRC.